EngineLayer/GlycoSearch/GlycanBox.cs

```diff
@@ -0,0 +1,139 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Chemistry;
+using System;
+using Proteomics;
+using MassSpectrometry;
+
+namespace EngineLayer
+{
+    //GlycanBox -- a list of all glycans that could be on a peptide. The GlycanBoxMass is the total mass of all glycans on the peptide
+    public class GlycanBox:ModBox
```

Comment:
maybe rename to `GlycanPossibilities`?

Reply:
I prefer the orginal name. Possiblilities is more confuse. I will comment it well.